### PR TITLE
Updated empty states when image scanning is disabled

### PR DESCRIPTION
**Pull request by milindk8**
# Description

Update the image of empty states when image scanning is not enabled

# Motivation

The image depicts the process as in progress, which is not true. Hence, updating the images for a better user experience

Fixes - https://github.com/devtron-labs/devtron/issues/2007

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/EmptyState/emptyState.scss [hunk 1]
```diff
@@ -68,6 +68,11 @@
         text-align: center;
         color: var(--N900);
     }
+    .button__icon {
```
Comment (shivani170) [inline src/components/EmptyState/emptyState.scss:71]: why are we not using icon-dim-16 here?

File: src/components/app/details/cIDetails/CIDetails.tsx [hunk 2]
```diff
@@ -1129,6 +1129,8 @@ const SecurityTab: React.FC<{ triggerHistory: History }> = (props) => {
         isError: false,
     })
     const { appId } = useParams<{ appId: string }>()
+    const ciPipelineId = props?.triggerHistory?.ciPipelineId
```
Comment (shivani170) [inline src/components/app/details/cIDetails/CIDetails.tsx:1132]: rather initializing `ciPipelineId` globally. We must move it into `redirectToCreate`

File: src/components/app/details/cIDetails/CIDetails.tsx [hunk 3]
```diff
@@ -1161,6 +1163,11 @@ const SecurityTab: React.FC<{ triggerHistory: History }> = (props) => {
         }
     }, [props.triggerHistory.artifactId])
 
+    const redirectToCreate = () => {
+        let url = `${URLS.APP}/${appId}/${URLS.APP_CONFIG}/${URLS.APP_WORKFLOW_CONFIG}/${ciPipelineId}/${URLS.APP_CI_CONFIG}/${ciPipelineId}/build`
```
Comment (shivani170) [inline src/components/app/details/cIDetails/CIDetails.tsx:1167]: replace let to const

**Review by shivani170 — APPROVED** [review]

File: src/components/app/details/cIDetails/CIDetails.tsx (hunk 2 above)
Comment (milindk8) [reply src/components/app/details/cIDetails/CIDetails.tsx]: Done

File: src/components/app/details/cIDetails/CIDetails.tsx (hunk 3 above)
Comment (milindk8) [reply src/components/app/details/cIDetails/CIDetails.tsx]: Done

File: src/components/app/details/cIDetails/CIDetails.tsx [hunk 4]
```diff
@@ -1161,6 +1162,12 @@ const SecurityTab: React.FC<{ triggerHistory: History }> = (props) => {
         }
     }, [props.triggerHistory.artifactId])
 
+    const redirectToCreate = () => {
+        const ciPipelineId = props?.triggerHistory?.ciPipelineId
+        const url = `${URLS.APP}/${appId}/${URLS.APP_CONFIG}/${URLS.APP_WORKFLOW_CONFIG}/${ciPipelineId}/${URLS.APP_CI_CONFIG}/${ciPipelineId}/build`
```
Comment (vivek-devtron) [inline src/components/app/details/cIDetails/CIDetails.tsx:1167]: we can directly use the value in push no need to create variables

Comment (milindk8) [reply src/components/app/details/cIDetails/CIDetails.tsx]: Done

**Review by shivani170 — APPROVED** [review]

**Pull request merged**